config/credentials.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  GITHUB_CLIENT_ID: '',
-  GITHUB_CLIENT_SECRET: '',
+  GITHUB_CLIENT_ID: 'd52ee29c2e854e1d4cd6',
```

Comment:
We should not be committing our github client IDs and secrets to the repo

Reply:
actually this file should have been added to the `.gitignore` which would remove this file from source control. Can you try rebasing on the most recent master and updating your PR branch?

Reply:
I just deleted the info manually and committed